pxr/usd/sdf/testenv/testSdfBatchNamespaceEdit.py

```diff
@@ -31,6 +31,47 @@
 #verbose = True
 
 class TestSdfBatchNamespaceEdit(unittest.TestCase):
+    def test_EqualityOperators(self):
+        self.assertTrue(Sdf.NamespaceEdit('/A', '/B') == Sdf.NamespaceEdit('/A', '/B'))
+        self.assertFalse(Sdf.NamespaceEdit('/A', '/B') != Sdf.NamespaceEdit('/A', '/B'))
+        self.assertFalse(Sdf.NamespaceEdit('/B', '/A') == Sdf.NamespaceEdit('/A', '/B'))
+        self.assertTrue(Sdf.NamespaceEdit('/B', '/A') != Sdf.NamespaceEdit('/A', '/B'))
+
+        # Validate the equality operators by varying a single field from a prototype edit
+        # at a time
+        prototype = Sdf.NamespaceEditDetail(
+            Sdf.NamespaceEditDetail.Okay,
+            Sdf.NamespaceEdit('/A', '/B'),
+            "reason"
+        )
+        self.assertTrue(prototype == prototype)
```

Comment:
The prototype == prototype test case would be more robust if it were
self.assertTrue(prototype == Sdf.NamespaceEditDetail(
￼            Sdf.NamespaceEditDetail.Okay,
￼            Sdf.NamespaceEdit('/A', '/B'),
￼            "reason"
￼        )
Same for !=

Reply:
Done, I also switched the tests to use `assertEqual` and `assertNotEqual` which is preferred per https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertFalse